nickel/defs.bzl

```
@@ -147,8 +147,14 @@ above the execution root are not allowed and will result in an error.
 """,
     ),
     "deps": attr.label_list(
-        doc = "Nickel files required by the top-level file",
-        allow_files = [".ncl"],
+        doc = "Source files required by the top-level files",
+        allow_files = [
+            ".json",
+            ".ncl",
+            ".toml",
+            ".yaml",
+            ".yml",
```

```suggestion
            ".yml",
            ".txt",
```
Since with Nickel 1.4 we can import files with a `txt` extension as raw strings. In the future, I hope we'll remove this filename sensitivity in favor of a format specifier on the import statement.

---

Thank you.